### PR TITLE
#1569 - Metrics Side Menu, Current Page Highlights

### PR DESCRIPTION
**Pull request by kodinkat**
- #1569 

Screenshot 2022-01-26 at 16 45 40

Screenshot 2022-01-26 at 16 45 05

Screenshot 2022-01-26 at 16 44 47

### CODE REVIEW

File: dt-users/user-management.php [hunk 1]
```diff
@@ -15,7 +15,7 @@ public static function instance() {
     public function __construct() {
         $url_path = dt_get_url_path();
         if ( self::has_permission() || self::non_admins_can_make_users() ) {
-            if ( strpos( $url_path, 'user-management/user' ) !== false || ( strpos( $url_path, 'user-management/add-user' ) !== false && ( current_user_can( "create_users" ) || self::non_admins_can_make_users() ) ) ){
+            if ( strpos( $url_path, 'user-management/user' ) !== false || strpos( $url_path, 'user-management/hover-map' ) !== false || ( strpos( $url_path, 'user-management/add-user' ) !== false && ( current_user_can( "create_users" ) || self::non_admins_can_make_users() ) ) ){
```
Comment (corsacca) [inline dt-users/user-management.php:18]: @kodinkat Lets find a better strategy for when we add more user management pages.

**Review by corsacca — CHANGES_REQUESTED** [review]

File: dt-users/user-management.php (hunk 1 above)
Comment (kodinkat) [reply dt-users/user-management.php]: @corsacca As `Hover Map` template is shared across metrics and user-management views, decided to shift away from class name, but instead; based conditional logic on `url_path` structure...

File: dt-users/user-management.php [hunk 2]
```diff
@@ -15,16 +15,16 @@ public static function instance() {
     public function __construct() {
         $url_path = dt_get_url_path();
         if ( self::has_permission() || self::non_admins_can_make_users() ) {
-            if ( strpos( $url_path, 'user-management/user' ) !== false || ( strpos( $url_path, 'user-management/add-user' ) !== false && ( current_user_can( "create_users" ) || self::non_admins_can_make_users() ) ) ){
+            if ( str_contains( $url_path, 'user-management/' ) && ( current_user_can( "create_users" ) || self::non_admins_can_make_users() ) ){
```
Comment (corsacca) [inline dt-users/user-management.php:18]: @kodinkat is this part still needed?

Comment (kodinkat) [reply dt-users/user-management.php:18]: @corsacca I believe so; so as to future-proof against addition of any new page sections..

Comment (corsacca) [reply dt-users/user-management.php:18]: @kodinkat the js part that we would need is no longer there.

Comment (corsacca) [reply dt-users/user-management.php:18]: Oh. I mean for line 128

Comment (corsacca) [reply dt-users/user-management.php:18]: and 24 i think

**Review by corsacca — APPROVED** [review]

**Pull request merged**